contents/tutorials/new-user-experiments.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Running Experiments on new users 
+sidebar: Docs
+showTitle: true
+author: ['ian-vanagas']
+date: 2022-10-10
+featuredImage: ../images/tutorials/banners/new-user-experiments.png
+topics: ['experiments', 'feature flags']
+---
+
+- **Level:** Medium 🦔🦔
+- **Estimated reading time:** 10 minutes ☕️☕️
+
+Optimizing the initial experience of new users is critical for turning them into existing users. Products have a limited amount of time and attention from new users before they leave and churn. 
+
+One way PostHog helps create better new user experiences is through Experiments. Experiments help teams test changes and their effect on key metrics to create better products. 
+
+Experiments start with a proposed change and hypothesized impact. They then guide you through setting up the user groups and feature flags for testing. Finally, they provide analysis of results. 
+
+We’ll use all this to help us create a better new user experience.
+
+## Our experiment and what we need
+
+For our experiment, we want to see if a change to our onboarding process gets more new users to complete the first key event in our product, uploading a file. In our example, we have design and copy changes we’d to try to encourage new users to upload a file.
+
+We’ve already set up PostHog ([on a paid plan](/pricing), to get access to Experiments), and have custom events set up for both `signed_up` and `uploaded_file`. Both have a significant amount of events coming in. 
+
+We’ll also need a `joined_at` user property. Autocapture does not capture this detail so we’ll need to set it up somewhere in our code. We can use `$set_once` in the event capture for `signed_up` to make this happen (find more details on setting [user properties here](/docs/integrate/client/js#set_once))
+
+This is all we need to start setting up our experiment.
+
+## Setting up our experiment in PostHog
+
+To start setting up our experiment, go to the Experiments tab, and start entering basic info like a  name, key, and description. We are going to test two design and copy changes so we’ll create variant keys for both. One is named `upload-prompt`, and the other is `upload-instructions`.
+
+![New experiment](../images/tutorials/new-user-experiments/new-experiment.png)
+
+For our participants, we will choose users, and then use regex to filter for ones who joined recently. We’ll use the regex equation `2022-09.+` to get filter all the users who joined in September (because that was last month).
```

Comment:
```suggestion
For our participants, we will choose users, and then use regex to filter for ones who joined recently. We’ll use the regex equation `2022-09.+` to filter all the users who joined in September (because that was last month).
```